content/en/docs/rules/conditions.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Condition Syntax
+weight: 3
+---
+
+A condition is a boolean expression related to a single event that has been detected by Falco. You can use fields related to every supported event, but we'll focus on syscalls as they're currently the most common. The language supports boolean operators and parentheses as you'd expect. For example a condition like:
+
+```
+evt.type = execve and evt.dir = < and (proc.name = cat or proc.name = grep)
+```
+
+Will trigger for each execution of `cat` or `grep`. We'll now take a closer look at what is the meaning of those fields such as `evt.dir`, how to discover the types of events there are available and which fields go with which event type.
```

Comment:
This will trigger for each execution of `cat` or `grep`.You can take a closer look at what is the meaning of those fields such as `evt.dir`, how to discover the types of events available, and which fields go with which event type.